guides/release-process.md

```diff
@@ -134,24 +135,18 @@ In the following instructions, "X.Y.Z" is used to denote the [next version of Cy
     - Create a new patch release (and a new minor release, if this is a minor release) in ZenHub, and schedule them both to be completed 2 weeks from the current date.
     - Move all issues that are still open from the current release to the appropriate future release.
 
-17. Bump `version` in [`package.json`](package.json), commit it to `develop`, tag it with the version, and push the tag up:
+17. Bump `version` in [`package.json`](package.json), submit, get approvals on, and merge a new PR for the change. After it merges:
 
     ```shell
-    git commit -am "release X.Y.Z [skip ci]"
+    git checkout develop
+    git pull origin develop
     git log --pretty=oneline
     # copy sha of the previous commit
     git tag -a vX.Y.Z <sha>
     git push origin vX.Y.Z
     ```
 
-18. Merge `develop` into `master` and push both branches up. Note: pushing to `master` will automatically publish any independent npm packages that have not yet been published.
-
-    ```shell
-    git push origin develop
-    git checkout master
-    git merge develop
-    git push origin master
-    ```
+18. Submit, get approvals on, and merge a new PR that merges `develop` to `master`
```

Comment:
Is step 18 redundant to step 17?

Reply:
Step 17 is updating develop. Step 18 is pulling those changes into master.